src/iodined.c

```diff
@@ -1597,6 +1597,27 @@ handle_a_request(int dns_fd, struct query *q, int fakeip)
 	}
 }
 
+static void
+handle_underscore_request(int dns_fd, struct query *q, const char *topdomain)
+{
+	char buf[64*1024];
+	int len;
+
+	len = dns_encode_nxdomain(buf, sizeof(buf), q, topdomain);
+	if (len < 1) {
+		warnx("ens_encode_a_response doesn't fit");
```

Comment:
Not matching the encode function name (also dns was changed to ens?)

Reply:
Yes, some ctrl+c,ctrl+v without attention. Sorry